### PR TITLE
Add callout when using existing Cognito Resource

### PR DESCRIPTION
**Pull request by nadetastic**
_Issue #, if available:_
https://github.com/aws-amplify/amplify-js/issues/3526

_Description of changes:_
Add a callout for when using existing Cognito resources. If UserPool client doesn't have required attributes set to mutable, then login failure will happen with Social Sign In

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/lib/auth/js/start.mdx [hunk 1]
```diff
@@ -127,6 +127,9 @@ Amplify.configure({
 // You can get the current config object
 const currentConfig = Auth.configure();
 ```
+<Callout warning={true}>
```
Comment (abdallahshaban557) [inline src/fragments/lib/auth/js/start.mdx:130]: Is there a link we can point customers to on how to do that? maybe from the Cognito docs?

Comment (nadetastic) [reply src/fragments/lib/auth/js/start.mdx:130]: Yes, this looks sufficient - https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html

I'll add and update

**Review by abdallahshaban557 — APPROVED** [review]
LGTM!

**Review by cwomack — APPROVED** [review]
LGTM with the addition of the reference link!

**Pull request merged**